core/netparams/defaults.go

```diff
@@ -162,13 +162,13 @@ func defaultNetParams() map[string]value {
 		StakingAndDelegationRewardOptimalStakeMultiplier:  NewDecimal(gteD1).Mutable(true).MustUpdate("3.0"),
 
 		// spam protection policies
-		SpamProtectionMaxVotes:               NewInt(gteI1).Mutable(true).MustUpdate("3"),
-		SpamProtectionMinTokensForVoting:     NewDecimal(gteD1).Mutable(true).MustUpdate("100000000000000000000"),
-		SpamProtectionMaxProposals:           NewInt(gteI1).Mutable(true).MustUpdate("3"),
-		SpamProtectionMinTokensForProposal:   NewDecimal(gteD1).Mutable(true).MustUpdate("100000000000000000000000"),
-		SpamProtectionMaxDelegations:         NewInt(gteI1).Mutable(true).MustUpdate("390"),
-		SpamProtectionMinTokensForDelegation: NewDecimal(gteD1).Mutable(true).MustUpdate("1000000000000000000"),
-
+		SpamProtectionMaxVotes:                         NewInt(gteI1).Mutable(true).MustUpdate("3"),
+		SpamProtectionMinTokensForVoting:               NewDecimal(gteD1).Mutable(true).MustUpdate("100000000000000000000"),
+		SpamProtectionMaxProposals:                     NewInt(gteI1).Mutable(true).MustUpdate("3"),
+		SpamProtectionMinTokensForProposal:             NewDecimal(gteD1).Mutable(true).MustUpdate("100000000000000000000000"),
+		SpamProtectionMaxDelegations:                   NewInt(gteI1).Mutable(true).MustUpdate("390"),
+		SpamProtectionMinTokensForDelegation:           NewDecimal(gteD1).Mutable(true).MustUpdate("1000000000000000000"),
+		SpamProtectionMinimumWithdrawalQuantumMultiple: NewDecimal(DecimalGT(num.MustDecimalFromString("0")), DecimalLT(num.MustDecimalFromString("10e6"))).Mutable(true).MustUpdate("10"),
```

Comment:
SpamProtectionMinimumWithdrawalQuantumMultiple: NewDecimal(gt0, DecimalLT(num.MustDecimalFromString("10e6"))).Mutable(true).MustUpdate("10"),

also, is this meant to be 1e7? or did you mean 1e6?

Reply:
This meant to be "10^6" per what Klaus asked me?

Reply:
10^6 is 1e6 not 10e6